### ISSUE TITLE
[wasm][eat] Failing test because of "Use of ResourceManager for custom types is disabled"

### ISSUE THREAD

**Issue by radical** [dotnet/runtime#51833]
[Log](https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-51697-merge-d7050d97e17f4af48c/System.Resources.Reader.Tests/console.f3db50e3.log?sv=2019-07-07&se=2021-05-14T10%3A12%3A51Z&sr=c&sp=rl&sig=joFjNEgxpTnETLo%2BQpy0GG9W3JRhjUYwDz51sJW1Qcs%3D)

```
[10:34:52] fail: [FAIL] System.Resources.ResourceWriterTests.ResourceReaderTests.ReadV1Resources
[10:34:52] info: System.NotSupportedException : Use of ResourceManager for custom types is disabled. Set the MSBuild Property CustomResourceTypesSupport to true in order to enable it.
[10:34:52] info:    at System.Resources.ResourceReader.FindType(Int32 typeIndex)
[10:34:52] info:    at System.Resources.ResourceReader._LoadObjectV1(Int32 pos)
[10:34:52] info:    at System.Resources.ResourceReader.LoadObjectV1(Int32 pos)
[10:34:52] info:    at System.Resources.ResourceReader.LoadObject(Int32 pos)
[10:34:52] info:    at System.Resources.ResourceReader.ResourceEnumerator.get_Entry()
[10:34:52] info:    at System.Resources.ResourceReader.ResourceEnumerator.get_Current()
[10:34:52] info:    at System.Resources.ResourceWriterTests.ResourceReaderTests.ReadV1Resources()
[10:34:52] info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
```

**Comment by marek-safar** [comment]
This is a test issue where the test should be conditional on https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/libraries/System.Private.CoreLib/src/System/Resources/ResourceReader.cs#L131

**Comment by MichalStrehovsky** [comment]
Fixed in #72497
